blog/docs/articles/from-cloudevent-to-apach-kafka-records-part-one.md

```diff
@@ -0,0 +1,116 @@
+# From CloudEvents to Apache Kafka Records, Part I
+
+**Authors: Daniele Zonca, Senior Principal Software Engineer @ Red Hat, Matthias Weßendorf, Principal Software Engineer @ Red Hat**
+
+**Date: 2023-03-08**
+
+_In this blog post you will learn how to easily store incoming CloudEvents to an Apache Kafka Topic using the KafkaSink component._
+
+
+Apache Kafka is used in a lot of very different use cases but the need to adopt Kafka protocol can be a barrier especially when there are third party components with limited extension possibilities.
+
+There are producer of events that doesn't support Kafka protocol and HTTP can be a more flexible option. Strimzi project has a [Bridge component](https://strimzi.io/docs/bridge/latest/) that exposes producer/consumer API via HTTP but it is specific for Kafka so it is essentially the same protocol (with consumer group, offset, etc).
```

Comment:
I'm not an English expert but shouldn't this be "There are producers of events that don't" ?

Reply:
yeah, even `don't` is not that proper I recall :-)   I will update, thanks @pierDipi

Reply:
what about producer -> producers?